src/api/comments.ts

```diff
@@ -0,0 +1,21 @@
+import { request } from './api';
+
+export const getPostComments = (postId: number) => {
+  return request(`/comments?postId=${postId}`);
+};
+
+export const deletePostComment = (commentId: number) => {
+  return request(`/comments/${commentId}`, {
+    method: 'DELETE',
+  });
+};
+
+export const postNewComment = (comment: PostComment) => {
+  return request('/comments', {
+    method: 'POST',
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+    body: JSON.stringify(comment),
+  });
+};
```

Comment:
this method looks fine to me, can't say what's wrong for now, maybe try to debug to see what is returned from fetch